README.md

```diff
@@ -7,8 +7,25 @@
 
 
 ## Usage
+### trace_generator.py
+This script genrates a dummy trace in binary and saves it in "resources/bin" as a csv.
+#### Execute from command line:
+````
+python3 trace_generator.py
+````
+
+### trace_analysis_main.py
+This script runs all necessary sub-scripts for filtering all type of write commands. Saves the filtered trace to "resources/filtered_trace".
+When executing the script you can set multiple flags for some options.
+* --logger_level [ERROR, INFO or DEBUG]: ERROR = no logs, INFO = basic info (default), DEBUG = ultimate info
+* --hex: uses already converted hex files for analyzing, if this flag is not set it uses a trace in binary and converts it to hex
+* --fname [filename]: Filename of trace which should be analyzed
```

Comment:
You have already described the flags with argparse - so remove the part in README and provide a `--help` command instead